### DISCUSSION TITLE
希望新增译文覆盖原文的功能

### DISCUSSION THREAD

**Discussion by RemyYYZ** [immersive-translate/immersive-translate#441] [Ideas]
### 新功能 | New Feature

新增译文覆盖原文的功能

### 为什么您想要添加这些功能 | Why you want to add these features

在一些网页元素密集的网站中，原文和译文可能会相互覆盖，导致看不清楚。

### 提议 | Proposal

建议在页面设置中的译文显示样式里添加译文覆盖原文的选项。

**Comment by versun** [comment]
会做这个功能的哈，关注更新

**Comment by versun** [comment]
0.5.7版本已添加该功能, 火狐浏览器已上架更新, 其他平台的还在审核
点击[更多] -> [切换为仅显示译文]

**Reply by RemyYYZ** [reply]
用户建议说到做到，而且办事效率很高，赞一个👍
